### PR TITLE
Add text label to copy buttons on tags

### LINKED ISSUE #1506: [Feature]: Replace Copy icons with text buttons
In #1505 we added a second copy icon to allow copying tag value, not always full JSON.

image

It would increase usability if instead of the icons we used plain text buttons "Copy" and "JSON".

### PR DESCRIPTION
**Pull request by Yushmanth-reddy**
## Which problem is this PR solving?
-  Resolves #1506 

## Short description of the changes
- Previously there we only have icons to copy text and JSON. But now we have implemented plain text along with the icons.
Screenshot from 2023-07-06 13-23-12

### CODE REVIEW

**Review by yurishkuro — DISMISSED** [review]
looks great!

File: packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/KeyValuesTable.tsx [hunk 1]
```diff
@@ -153,12 +153,14 @@ export default function KeyValuesTable(props: KeyValuesTableProps) {
                     className="KeyValueTable--copyIcon"
                     copyText={row.value}
                     tooltipTitle="Copy value"
+                    buttonText="Copy"
                   />
                   <CopyIcon
-                    className="KeyValueTable--copyIcon"
+                    className="KeyValueTable--copyIcon copyIcon"
```
Comment (yurishkuro) [inline packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/KeyValuesTable.tsx:159]: `copyIcon` - why is this needed? It's not even defined in the css file.

Comment (Yushmanth-reddy) [reply packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/KeyValuesTable.tsx:159]: Sorry my bad I thought to use it in css of keyValueTable.css but later realised to use the css of Copyicon.css. Finally it is of no use.Let me make the changes.

**Review by yurishkuro — APPROVED** [review]
